packages/demo-app/src/App.tsx

```diff
@@ -1,66 +1,80 @@
-import { LogtoProvider, useLogto } from '@logto/react';
+import { LogtoProvider, useLogto, UserInfoResponse } from '@logto/react';
 import { signInNotificationStorageKey } from '@logto/schemas';
 import { demoAppApplicationId } from '@logto/schemas/lib/seeds';
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 
 import '@/scss/normalized.scss';
 import * as styles from './App.module.scss';
 import Callback from './Callback';
 import congrats from './assets/congrats.svg';
+import initI18n from './i18n/init';
+
+void initI18n();
```

Comment:
never deep dive into this `i18nInit` logic, always wondering could it possible that the component rendered before i18n is initialized?

Reply:
I would say yes? The react component rendering will not be affected by i18n initialization, but the text will not show i18n contents, and it would end up using the phrase keys instead